README.md

```diff
@@ -15,6 +15,7 @@ Kubernetes interfaces.
 
 * [Metal³ Development Mailing List](https://groups.google.com/forum/#!forum/metal3-dev)
 * [#cluster-api-baremetal](https://kubernetes.slack.com/messages/CHD49TLE7) on Kubernetes Slack
+* Bi-Weekly Every alternate Wednesday at 14:00UTC (Join Zoom Meeting https://zoom.us/j/781102362 )
```

Comment:
@sb1975 Try to use same format as the other links, it is:
[Text to describe the link](link to the resource)
And also point that is a community meeting in the description.
Thank you!